components/AddTask.js

```diff
@@ -1,26 +1,25 @@
-import React from 'react';
-import { View, StyleSheet, TextInput } from "react-native";
+import React, { useState } from 'react';
+import { View, StyleSheet, TextInput, Text } from "react-native";
+import DateTimePicker from '@react-native-community/datetimepicker';
 import { Button } from "react-native-elements";
 
 class AddTask extends React.Component {
   constructor() {
     super();
     this.state = {
       name: '',
-      time: '',
-      window: '',
+      time: new Date(),
```

Comment:
Does this set `this.state.date` to the current time/date?

Reply:
I believe so. And now that you mention it it's not quite the correct syntax. I needed this to start out as a date for the DateTimePicker to work, but they're only conditionally rendered based onFocus of their respective TextInput. This should be okay to continue, but I will make an issue to refactor.